reana_client/cli/files.py

```diff
@@ -555,7 +607,7 @@ def workflow_disk_usage(
 ):  # noqa: D301
     """Get workspace disk usage.
 
-    The ``du`` command allows to chech the disk usage of given workspace.
+    The ``du`` command allows to check the disk usage of given workspace.
```

Comment:
Please fix the same typo also in `reana-client-go` repository.  (But we can also do it later when porting the command to Golang client.)

Also, `docs.reana.io` client CLI API would need updating.